### PR TITLE
Adds some backend admin stuff

### PR DESCRIPTION
**Pull request by Ty-Omaha**
## What Does This PR Do
Adds some backend stuff for admins
This requires a server sided script to work.
Requires DB update

🆑 
add: Adds some backend admin stuff
/ 🆑

### CODE REVIEW

**Comment by Ty-Omaha** [comment]
@Kyep

File: code/modules/admin/ipintel.dm [hunk 1]
```diff
@@ -225,4 +225,68 @@
...
+							if(1)//user is dirty
+								message_admins("<span class='warning'>[alert_message]</span>")
+								log_game("[alert_message]")
+
+								if(!dbcon.IsConnected() || !config.sql_enabled) //exit if the local database isnt working
+									return
+								message_admins("STEP 1")
+								var/sql = "SELECT ckey FROM [format_table_name("high_risk")] WHERE ckey = '[sanitizeSQL(ckey)]'"
+								var/DBQuery/query = dbcon.NewQuery(sql)
+								message_admins("STEP 2: [sql]")
```
Comment (Ty-Omaha) [inline code/modules/admin/ipintel.dm]: need to remove debug lines

File: code/modules/admin/ipintel.dm [hunk 2]
```diff
@@ -225,4 +225,68 @@
...
+/proc/high_risk_check(var/ip, var/client/C)
+	if(!config.high_risk_checks)
+		return
+	var/ckey = C.ckey
+	if(C.player_age == "Requires database")
+		return
+	if(vpn_whitelist_check(ckey))
+		return 
+	if(C.player_age <= config.high_risk_age)
+		var/list/http[] = world.Export("http://127.0.0.1/hrcheck/check.php?requestedIP=[sanitizeSQL(ip)]") //MAKE SURE PATH IS CORRECT BEFORE MERGING
```
Comment (Ty-Omaha) [inline code/modules/admin/ipintel.dm]: this must match the path of the file on the server

File: code/modules/admin/ipintel.dm [hunk 3]
```diff
@@ -225,4 +225,68 @@
...
+									add_note(ckey, "[alert_message]", null, "System", 0)
+									message_admins("STEP 4")
+							if(2)//oof something went wrong
+								handle_high_risk_error(C, ip, "Flag Status Failure: 2")
+							else
+								handle_high_risk_error(C, ip, "Flag Rating Failure")
+					else
+						handle_high_risk_error(C, ip, "Isnum Failure")
+				else
+					handle_high_risk_error(C, ip, "Response Failure")
```
Comment (Ty-Omaha) [inline code/modules/admin/ipintel.dm]: This probably doesn't need to message_admins

File: code/modules/admin/ipintel.dm [hunk 4]
```diff
@@ -225,4 +225,68 @@
...
+	var/ckey = C.ckey
+	if(C.player_age == "Requires database")
+		return
+	if(vpn_whitelist_check(ckey))
+		return 
+	if(C.player_age <= config.high_risk_age)
+		var/list/http[] = world.Export("http://127.0.0.1/hrcheck/check.php?requestedIP=[sanitizeSQL(ip)]") //MAKE SURE PATH IS CORRECT BEFORE MERGING
+
+		if(http)
+			var/status = text2num(http["STATUS"])
```
Comment (AffectedArc07) [inline code/modules/admin/ipintel.dm]: Isnt this lowercase

Comment (Ty-Omaha) [reply code/modules/admin/ipintel.dm]: https://github.com/ParadiseSS13/Paradise/blob/6582f22be5f5565a53ee496d9a551b93fdc076f5/code/modules/admin/ipintel.dm#L88

File: SQL/updates/9-10.sql [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+#Adding new table for enhanced ban evasion detection
+#Adding ckey as the PK and boolean for logic in the code
+
+CREATE TABLE `high_risk` (
+  `ckey` varchar(45) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+ALTER TABLE `high_risk`
+  ADD PRIMARY KEY (`ckey`);
+COMMIT;
```
Comment (AffectedArc07) [inline SQL/updates/9-10.sql]: Shouldnt the primary key be a default thing, not an addition to the table you just made?

File: code/modules/admin/ipintel.dm (hunk 2 above)
Comment (AffectedArc07) [inline code/modules/admin/ipintel.dm]: This feels like it should be a config option anyways

**Comment by Ty-Omaha** [comment]
kyet doesnt want this

**Pull request closed without merging**